.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    docker:
+        - image: kaikuehne/pyenv:alpine3.10
```

Comment:
We should probably have our own version of this image in the startupgrind org.  (eventually)